docs/includes/file-get-started-excel-vue.md

```diff
@@ -16,131 +18,64 @@ In this article, you'll walk through the process of building an Excel add-in usi
-- **Setup e2e tests with Nightwatch?** `No`
+### Create the web app
 
-![Vue CLI prompts](../images/vue-cli-prompts.png)
+1. Create a folder on your local drive and name it **my-addin**. This is where you'll create the files for your app.
 
-## Generate the manifest file
-
-Each add-in requires a manifest file to define its settings and capabilities.
-
-1. Navigate to your app folder.
+2. Navigate to your app folder.
 
     ```bash
-    cd my-add-in
+    cd my-addin
     ```
 
-2. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown below.
+3. Use the Yeoman generator to generate the manifest file for your add-in. Run the following command and then answer the prompts as shown in the following screenshot.
 
     ```bash
-    yo office 
+    yo office
     ```
 
-    - **Choose a project type:** `Office Add-in containing the manifest only`
+    - **Choose a project type:** `Office Add-in project using Vue framework`
     - **What do you want to name your add-in?:** `My Office Add-in`
     - **Which Office client application would you like to support?:** `Excel`
 
-    After you complete the wizard, a manifest file and resource file are available for you to build your project.
-
-    ![Yeoman generator](../images/yo-office.png)
-    
-    > [!NOTE]
-    > If you're prompted to overwrite **package.json**, answer **No** (do not overwrite).
-
-## Secure the app
-
-[!include[HTTPS guidance](../includes/https-guidance.md)]
-
-To enable HTTPS for your app, open **package.json** in the root of the project, modify the `dev` script to add the `--https` flag, and save the file.
-
-```json
-"dev": "webpack-dev-server --https --inline --progress --config build/webpack.dev.conf.js"
-```
-
-## Update the app
-
-1. In your code editor, open the manifest file (i.e., the file in the root directory of your app with a name ending in "manifest.xml"). Replace all occurrences of `https://localhost:3000` with `https://localhost:8080` and save the file.
-
-2. Open **index.html**, add the following `<script>` tag immediately before the `</head>` tag, and save the file.
-
-    ```html
-    <script src="https://appsforoffice.microsoft.com/lib/1/hosted/office.js"></script>
-    ```
-
-3. Open **src/main.js** and *remove* the following block of code:
-
-    ```js
-    new Vue({
-        el: '#app',
-        components: {App},
-        template: '<App/>'
-    })
-    ```
+    ![Yeoman generator](../images/yo-office-excel-vue.png)
     
-    Then add the following code in that same location, and save the file. 
-                                                         
-    ```js
-    const Office = window.Office
-    Office.initialize = () => {
-      new Vue({
-        el: '#app',
-        components: {App},
-        template: '<App/>'
-      })
-    }
-    ```
+    After you complete the wizard, the generator will create the project and install supporting Node components.
```

Comment:
"the generator will create the project and install" -> "the generator creates the project and installs"